core/application/react-query-adapter/hackathon/client/use-get-hackathons-projects-v2.ts

```diff
@@ -0,0 +1,27 @@
+import { useInfiniteQuery } from "@tanstack/react-query";
+
+import {
+  UseInfiniteQueryFacadeParams,
+  useInfiniteQueryAdapter,
+} from "@/core/application/react-query-adapter/helpers/use-infinite-query-adapter";
+import { bootstrap } from "@/core/bootstrap";
+import { GetHackathonProjectsV2Model } from "@/core/domain/hackathon/hackathon-contract.types";
+import { HackathonFacadePort } from "@/core/domain/hackathon/inputs/hackathon-facade-port";
+import { HackathonStoragePort } from "@/core/domain/hackathon/outputs/hackathon-storage-port";
+
+export function useGetHackathonProjectsV2({
```

Comment:
```suggestion
export function useGetHackathonProjects({
```

No need to call this V2 as V1 doesn't exist ^^